cea/optimization/master/cost_model.py

```diff
@@ -211,15 +211,15 @@ def addCosts(indCombi, buildList, locator, dicoSupply, QUncoveredDesign, QUncove
         print pv.calc_Cinv_pv(PV_peak), "PV peak"
         
         SC_area = dicoSupply.SOLAR_PART_SC * solarFeat.SolarAreaSC
-        SCInvC = stc.calc_Cinv_SC(SC_area)
+        SCInvC = stc.calc_Cinv_SC(SC_area, gv)
```

Comment:
why is it calling gv? if the values should not be there, it should have called the database.